### PR TITLE
Handle community estabs

### PR DESCRIPTION
**Pull request by LukeLoze1**
Adds CE to sample file generation and validation
Adds the CE types

Run the tests. 'make test'

### CODE REVIEW

File: generate_sample_file.py [hunk 1]
```diff
@@ -95,6 +106,21 @@ def get_random_post_code(self):
...
+    def get_random_ce_capacity():
+        random_ce = random.randint(5, 1000)
+        return f'{random_ce}'
+
+    def get_random_ce_type(self):
+        return self.CE_TYPES[random.randint(0, len(self.CE_TYPES) - 1)]
+
+    @staticmethod
+    def random_is_community_estab():
+        if random.randint(0, 10) > 9:
```
Comment (liamtoozer) [inline generate_sample_file.py]: Could probably just `return random.randint(0, 10) > 9`, or maybe `return random.random() < 0.1`?

**Review by liamtoozer — APPROVED** [review]
LGTM 👍 Just a minor comment

File: generate_sample_file.py (hunk 1 above)
Comment (LukeLoze1) [reply generate_sample_file.py]: good idea, done

File: Pipfile [hunk 2]
```diff
@@ -13,6 +13,8 @@ codecov = "*"
 pika = "*"
 jinja2 = "*"
 redis = "*"
+python = "*"
+generate-sample-file-py = "*"
```
Comment (ryangrundy7) [inline Pipfile]: Did you mean to add these?

**Review by ryangrundy7 — APPROVED** [review]

**Pull request merged**